build-mbl/poky-build/build-poky.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python3
```

Comment:
I see also this one is taken from thud-gatweay-branch, isn't it?

Reply:
Initially from thud-gateway branch, but modified to suit the Poky build.